AppCreationScripts/AppCreationScripts.md

```diff
@@ -0,0 +1,107 @@
+# Registering the Azure Active Directory applications and updating the configuration files for this sample using PowerShell scripts
+
+## Goal of the scripts
```

Comment:
Does this script come from the same source as all the other samples?

Reply:
yes. Why?